astropy/io/ascii/basic.py

```diff
@@ -129,6 +129,8 @@ class CommentedHeader(Basic):
     for the last commented line).  The default delimiter is the <space>
     character.
 
+    This matches the format produced by ``np.savetxt()``, with ``delimiter=','``.
```

Comment:
The first time I saw this I assumed that it applied to structured arrays and would do the right thing, but no.  Then I blindly tried with a 2-d array and still didn't get the header column.  (Indeed I rarely use `np.savetxt()`).  So to avoid actually adding to confusion for folks like me, I would suggest something along the lines of:
```
This matches the format produced by ``np.savetxt()``, with ``delimiter=','``
and ``header='<comma-delimited-column-names-list>'``.
```

Reply:
@parejkoj - can you make that little edit so we can merge this?

Reply:
Thanks for the reminder. I think I missed your message above. Rebased to master and pushed.